libp2p-peerstore/README.md

```diff
@@ -0,0 +1,47 @@
+The `peerstore` crate allows one to store information about a peer.
+
+`peerstore` is a key-value database, where the keys are multihashes (which usually corresponds
+to the hash of the public key of the peer, but that is not enforced by this crate) and the
+values are the public key and a list of multiaddresses. Contrary a regular key-value storage,
```

Comment:
This is not contrary to how many KV stores work. Redis has TTL and it's a pretty common feature in other KV stores as far as I know.